PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/Annotation.java

```diff
@@ -120,7 +121,12 @@ public String buildMetaDataString() {
     sb.append("Age\t\t\t\t\t\t").append(age).append('\n');
     sb.append("Capreomycin\t\t\t\t").append(capreomycin).append('\n');
     sb.append("Cohort\t\t\t\t\t").append(cohort).append('\n');
-    sb.append("Date of collection\t\t\t").append(dateOfCollection).append('\n');
+    sb.append("Date of collection\t\t\t");
+    if (dateOfCollection != null) {
```

Comment:
Both branches are one line long. It is better practice/easier to read to read a positive if-condition than a negative. This is a [style rule imposed by PMD](http://pmd.sourceforge.net/pmd-4.3.0/rules/design.html#ConfusingTernary) (also in our project).